### PR TITLE
Add configurable severity level(s) and update argparse namespace

### PR DESCRIPTION
**Pull request by whardier**
This patch adds in severity level arguments and alters the graphql template for github actions.  It assumes that the user knows the proper levels and adding new providers may require setting up mappings against a strict enum choice or some such.

### CODE REVIEW

File: src/security_constraints/github_security_advisory.py [hunk 1]
```diff
@@ -15,13 +15,12 @@
...
 LOGGER = logging.getLogger(__name__)
 
-
 QUERY_TEMPLATE = string.Template(
     "{"
     "securityVulnerabilities("
     " first: $first"
     " ecosystem:PIP"
-    " severities:$severities"
+    " severities:[$severities]"
```
Comment (bunny-therapist) [inline src/security_constraints/github_security_advisory.py:23]: Given that "severities" is a list of str, how does it work when it goes inside brackets here?

Also, are severities case sensitive? (I did not check.)

File: src/security_constraints/main.py [hunk 2]
```diff
@@ -203,7 +204,18 @@ def get_args() -> Any:
...
     )
-    return parser.parse_args()
+    parser.add_argument(
+        "--severities",
+        type=str,
+        action="store",
+        nargs="+",
+        default=["critical"],
+        help=(
+            "Vulnerability severities include."
```
Comment (bunny-therapist) [inline src/security_constraints/main.py:214]: This sentence is a bit weird. Maybe "to include"?

**Review by bunny-therapist — COMMENTED** [review]
Since no tests were modified, I assume the default behavior is preserved?

I do believe there should be some test(s) added for the new behavior.

**Comment by bunny-therapist** [comment]
Ultimately, I think the cli itself should enforce correct severity levels, but that can be a later feature.

(Also, sorry I did not see this PR until now.)

**Comment by bunny-therapist** [comment]
I will merge this and fix any issues I find. The graphql stuff works, but I will fix/add tests and alter the cli a bit to have a minimum severity instead of a list of severities (also pass severity levels when fetching vulnerabilities rather than to the api).
